code/datums/diseases/transformation.dm

```diff
@@ -243,9 +243,15 @@
 		if(3)
 			if(ishuman(affected_mob))
 				var/mob/living/carbon/human/human = affected_mob
-				if(human.dna.species.id != SPECIES_SLIMEPERSON && affected_mob.dna.species.id != SPECIES_STARGAZER && affected_mob.dna.species.id != SPECIES_LUMINESCENT)
+				if(human.dna.species.id != SPECIES_MONKEY && human.dna.species.id != SPECIES_SLIMEPERSON && affected_mob.dna.species.id != SPECIES_STARGAZER && affected_mob.dna.species.id != SPECIES_LUMINESCENT)
```

Comment:
ismonkey isslimeperson

Reply:
isjellyperson actually to include stargazers and luminescents but :+1: